workspaces/orchestrator/README.md

```diff
@@ -1,115 +1,53 @@
+## Architecture Overview
 
-## Capabilities
+The architecture adheres to standard Backstage plugin guidelines and requires the following plugins to be installed for proper functionality:
 
-**Advanced core capabilities**
+1. **`backstage-plugin-orchestrator`**
 
-- Stateful/long-lived
-- Branching and parallelism
-- Error management and compensation
-- Event-driven supporting [CloudEvents](https://cloudevents.io)
-- Audit logging
-- Sub-flows
-- Choreography
-- Timer/timeout control
-- Built-in powerful expression evaluation with JQ
-- Low Code/No code
-- Cloud-native architecture Kubernetes/OpenShift with Operator support
-- OpenAPI / REST built-in integration etc.
+   - Provides the frontend interface.
 
-**Client-side tooling**
+2. **`backstage-plugin-orchestrator-backend`**
+   - Serves as a backend proxy between to SonataFlow.
 
-- Orchestration visualization / graphical editor
-- Integration with service catalog/actions
-- GitHub integration
-- Form generation
-- Runtime monitoring of instances
-- Dashboards
-- Potential custom integrations (user interaction, notifications, etc.)
+In addition to the main plugins, the architecture includes several isolated plugins that are imported by the above plugins:
 
-## For administrators
+1. **`backstage-plugin-orchestrator-swf-editor-envelope`**
 
-### Installation
+   - Hosts the workflow viewer.
 
-The Orchestrator plugin is composed of the following packages:
+2. **`backstage-plugin-orchestrator-common`**
 
-- `@red-hat-developer-hub/backstage-plugin-orchestrator-backend` package connects the Backstage server to the Orchestrator. For setup process, see [Backend Setup](#setting-up-the-orchestrator-backend-package)
-- `@red-hat-developer-hub/backstage-plugin-orchestrator` package contains frontend components for the Orchestrator plugin. For setup process, see [Frontend Setup](#setting-up-the-orchestrator-frontend-package)
-- `@red-hat-developer-hub/backstage-plugin-orchestrator-common` package contains shared code between the Orchestrator plugin packages.
+   - Contains the backend OpenAPI specification along with autogenerated API documentation and client libraries.
 
-#### Prerequisites for running the plugins locally in development mode
+3. **`backstage-plugin-orchestrator-form`**
 
-- Docker up and running
+   - Provides the workflow execution form.
 
-#### Setting up the Orchestrator as a dynamic plugin in a Helm deployment
+4. **`backstage-plugin-orchestrator-form-api`**
+   - Defines the API for extending the workflow execution form.
+   - For more details, see the [Extensible Form Documentation](./docs/extensibleForm.md).
 
-Please follow this link for instructions: https://github.com/janus-idp/backstage-showcase/blob/main/docs/dynamic-plugins/installing-plugins.md#installing-external-dynamic-plugins.
+## Install as a static plugin
```

Comment:
Why is this section needed?

Reply:
this is for users installing on vanilla backstage

Reply:
all RHDH plugins include this in the docs